cli/src/main/java/org/jboss/as/cli/handlers/trycatch/TryCatchFinallyControlFlow.java

```diff
@@ -57,6 +57,7 @@ static TryCatchFinallyControlFlow get(CommandContext ctx) {
     private int state;
 
     TryCatchFinallyControlFlow(CommandContext ctx) {
+        tryList = new ArrayList<>();
```

Comment:
doing so, any check for null in the class (I think there are some) should be removed.